charts/jenkins.io/templates/nginx-configmap.yaml

```
@@ -20,15 +20,26 @@ data:
       }
       {{- end }}
   
+      root   /usr/share/nginx/html;
+
       location / {
-          root   /usr/share/nginx/html;
           index  index.html index.htm;
           # Language setting
           if ($lang) {
             rewrite ^/$ https://jenkins.io/$lang$1;
           }   
       }
-  
+
+      location ~* \.(js|css|png|jpg|jpeg|gif|ico)$ {
```

Add the fonts in too.
Eventually add cors to the static assets too so plugin sites can use it instead of copying their own

https://coderwall.com/p/ad9ifw/nginx-aggressive-asset-configuration

```suggestion
      location ~* \.(?:css|js|woff|eot|svg|ttf|otf|png|gif|jpe?g) {
```